src/ucp/wireup/wireup_cm.c

```diff
@@ -269,7 +276,7 @@ ucp_cm_ep_priv_data_pack(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
 
     cm_idx = ucp_ep_ext_control(ep)->cm_idx;
     if (worker->cms[cm_idx].attr.max_conn_priv <
-        ucp_cm_priv_data_length(ucp_addr_size)) {
+        ucp_cm_priv_data_length(ucp_addr_size, consider_user_data)) {
```

Comment:
maybe no need the function ucp_cm_priv_data_length: just add local var "priv_data_length" in this function and initialize it once according to the needed logic